docs/Examples/Response.md

```diff
@@ -0,0 +1,118 @@
+# Response Handling
+
+You've now setup your TargetType correctly, and you've come to the fun part of implementing calls to your API.
+But one thing is not quite clear: How do you actually use the response from the `MoyaProvider`?
```

Comment:
I think its more readable and easier to understand this document when you are to the point stating the facts; not asking questions / filling in the user's thoughts. I've not seen this in other examples as well, so it feels a bit out of place.

Reply:
I think it's because I read the first line of [Basic.md](https://github.com/Moya/Moya/blob/master/docs/Examples/Basic.md) and then basically assumed that's how it's written. I can definitely be more factual. I'll have a look at how some of the other documentation is worded.